conf/prod.routes

```diff
@@ -3,4 +3,6 @@
 
 ->         /                                                     health.Routes
 
+->         /manage-transit-movements/departures                   app.transport.transportMeans.active.Routes
```

Comment:
Don't think this should be here. This can be removed and then transport means routes can be references from app.routes.

Reply:
It can be removed . It won't break anything as it will never be reached